conjur/config.py

```diff
@@ -29,25 +31,29 @@ class Config():
     FIELDS = [
         ('conjur_account', 'account', True),
         ('conjur_url', 'url', True),
-        ('cert_file', 'ca_bundle', False),
+        ('cert_file', 'ca_bundle', True),
     ]
 
     _config = {}
 
     def __init__(self, config_file=DEFAULT_CONFIG_FILE):
         # pylint: disable=logging-fstring-interpolation
-        logging.debug(f"Fetching connection details from filesystem {config_file}")
 
+        logging.debug(f"Fetching connection details from filesystem {config_file}")
         config = None
         with open(config_file, 'r') as config_fp:
             config = load(config_fp, Loader=Loader)
 
-        for config_field_name, attribute_name, mandatory in self.FIELDS:
-            if mandatory:
-                assert config_field_name in config
-
-            setattr(self, attribute_name, config[config_field_name])
-            self._config[attribute_name] = getattr(self, attribute_name)
+        if config:
```

Comment:
we talked earlier , and wanted to reduce nested if's. 
what many code analayzer suggest is to reduce the use of else.
```suggestion
        if not config: raise ConfigurationMissingException
```
and the rest of the code could run outside of the "if" statement. I think it will be well suited here